### ISSUE TITLE
Request: adding and saving locally subreddit like Flow for Reddit

### ISSUE THREAD

**Issue by JunkyardCat** [QuantumBadger/RedReader#432]
Hi 

how hard would it be to copy Flow for Reddit design? and its other functionality. I think the developer already abandoned it

is it possible to implement saving locally subreddit like Flow for Reddit? (not sure whats the proper name for that)

**Comment by QuantumBadger** [comment]
If you go to a subreddit, and open the three-dots menu, there should be an option called "Pin to Main Menu". I assume that does what you want?

**Comment by JunkyardCat** [comment]
yes that's it. I also found out that sidebar can also be found on that 3 dot menu. Quick question though is it hard to implement the "Flow for Reddit" UI design in redreader? im thinking of swiping right to left opens a portion of sidebar and left to right opens the menus
